src/meeting/meet/speakersObserver.ts

```
@@ -0,0 +1,675 @@
+                                    uniqueParticipants.set(uniqueKey, {
+                                        name: ariaLabel,
+                                        isSpeaking: false,
+                                        isPresenting: false,
+                                        isInMergedAudio: isMergedAudio,
+                                        cohortId: isMergedAudio ? cohortId : null,
+                                    })
+                                }
+
+                                const participant = uniqueParticipants.get(uniqueKey)!
+
+                                // Check if the participant is presenting
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)